app/javascript/packs/pack.js

```diff
@@ -1,40 +0,0 @@
-import { h, render } from 'preact';
```

Comment:
This has been renamed from pack.jsx to Onboarding.jsx as that us really what this pack file is for.